try块中声明的变量和catch块和finnaly块不在一个作用域